presto-kafka07/src/main/java/com/facebook/presto/kafka/KafkaConnectorConfig.java

```diff
@@ -179,15 +189,40 @@ public KafkaConnectorConfig setZkEndpoint(String zkEndpoint)
         return this;
     }
 
-    public int getFetchSize()
+    public DataSize getFetchSize()
     {
         return fetchSize;
     }
 
     @Config("kafka.fetch-size")
-    public KafkaConnectorConfig setFetchSize(int fetchSize)
+    public KafkaConnectorConfig setFetchSize(String fetchSize)
```

Comment:
I have a feeling you want to use the Builder pattern to achieve the immutable `Config` class. But it seems self-finished; not that great.